foundry/tensorflow_dataset_wrapper.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+from tensorflow.keras.utils import Sequence
+
+class FoundryDatasetAsTensorflowSequence(Sequence):
+    """Foundry Dataset Converted to Tensorflow Format"""
+
+    def __init__(self, inputs, targets):
+        self.inputs=inputs
+        self.targets=targets
+
+    def __len__(self):
+        return len(self.inputs[0])
+
+    def __getitem__(self, idx):
+        item = {"input": [], "target": []}
+        
+        for input in self.inputs:
+            item["input"].append(np.array(input[idx]))
+        item["input"] = np.array(item["input"])
+        
+        for target in self.targets:
+            item["target"].append(np.array(target[idx]))
+        item["target"] = np.array(item["target"])
+        
+        return item
```

Comment:
There is a newline missing at the end of the file, which is a requirement in Python's styleguide.

GH Actions hasn't been raising these style errors (see #208 ) but we should fix them anyway.